pkg/cloudinary/usage_report.go

```diff
@@ -52,6 +52,12 @@ func GetUsageReport(req *http.Request) (usageReport *UsageReport, err error) {
 	}
 	defer rs.Body.Close()
 
+	if rs.StatusCode != 200 {
```

Comment:
201 is valid for created resources. In this case it's unlikely to ever be returned (or we would have much bigger issues) but it could change in the future.

Reply:
Fixed!